src/datasets/synthetic.jl

```diff
@@ -60,6 +60,46 @@ function genZafarData(n = 10000, d = pi/4)
     return X, y
 end
 
+
+"""
+Generate synthetic data from Zafar et al., 2017 Fairness Beyond Disparate Treatment & Disparate Impact
+# Arguments
+- `n=10000` : number of samples
+# Returns
+- `X` : DataFrame containing features and protected attribute z
+- `y` : Binary Target variable
+"""
+using DataFrames
```

Comment:
Delete line 72 and 73. In Julia, you attach doc-strings to functions. So the triple quotes string should occur just before the function